### PR TITLE
Update unit tests to be consistent with upstream

### LINKED ISSUE #133: Correct cross-validation test harness `contract` and `address` and `operation` literals
Currently, cross-validation test harness and semantics uses address format  `@Address(...)` to represent address values in `address`, `contract`, and `operation` literals. Now that we have a better understanding of how this format works, we can fix the discrepancy with the upstream address format and do something much more reasonable.

References are found in:
- michelson.md
- tezos-client-unit-test
- several unit tests
- one coverage test

### PR DESCRIPTION
**Pull request by sskeirik**
Integrates changes from #37 into master (thanks @rafoo!)

Fixes #133.
Closes #37.

### CODE REVIEW

File: lib/tezos-client-unit-test [hunk 1]
```diff
@@ -67,7 +67,10 @@ fix_address() {
     export ORIGINATION_SUBS="$TEMP_DIR/origination_subs"
     export ALL_SUBS="$TEMP_DIR/subs"
 
-    if ! grep -o '@Address([^)"]*)' "$test" | sort | uniq > "$FOUND_ADDRESSES"; then
+    # -o - only matching
+    # -E - extended matches
+    # -e - pattern follows this option
+    if ! grep -o -E -e '"KT1[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{33}"' "$test" | tr -d '"' | sort | uniq > "$FOUND_ADDRESSES"; then
```
Comment (nishantjr) [inline lib/tezos-client-unit-test:73]:
```suggestion
    # This is the bitcoin base 58 alphabet, with "I", "O", "0" and "l" removed.
    if ! grep --only-matching --extended-regexp '"KT1[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{33}"' "$test" \
            | tr -d '"' | sort | uniq > "$FOUND_ADDRESSES"; then
```

Comment (nishantjr) [reply lib/tezos-client-unit-test:73]: We can remove the `if` by touching the file first, and then running the grep.

**Review by nishantjr — CHANGES_REQUESTED** [review]

File: lib/tezos-client-unit-test (hunk 1 above)
Comment (sskeirik) [reply lib/tezos-client-unit-test]: Addressed in later commit.

**Review by nishantjr — APPROVED** [review]
LGTM

**Pull request merged**